docs/04-operation-guides/troubleshooting/smsh-03-istio-no-sidecar.md

```diff
@@ -2,9 +2,14 @@
 title: Issues with Istio sidecar injection
 ---
 
+## Symptom & cause
 
-Kyma has sidecar injection enabled by default - a sidecar is injected to every Pod in a cluster without the need to add any labels. For more information, read [this document](#details-sidecar-proxy-injection).
-If a Pod doesn't have a sidecar and you did not disable sidecar injection on purpose, follow these steps to troubleshoot:
+Kyma has sidecar injection enabled by default - a sidecar is injected to every Pod in a cluster without the need to add any labels. For more information, read [this document](../operations/smsh-01-istio-disable-sidecar-injection.md).
```

Comment:
```suggestion
Kyma has sidecar injection enabled by default - a sidecar is injected to every Pod in a cluster without the need to add any labels. For more information, read the document about [disabling Istio sidecar proxy injection](../operations/smsh-01-istio-disable-sidecar-injection.md).
```

Reply:
Some time ago on one of SIG Content meetings we agreed to make links more descriptive and use titles/descriptions of the document's content instead of the phrase `read this document`.